src/tests/test_complete_dashboard.py

```
@@ -0,0 +1,8 @@
+import pytest
+
+def test_generate_plan_halts_on_inspection_failure(cli_runner, tmp_path):
+    # Simulate missing repository directory
+    missing_repo = tmp_path / "nonexistent"
+    result = cli_runner.invoke(main, ["generate-plan", "--repo-dir", str(missing_repo)])
```

Ruff (0.11.9)

Undefined name `main`

(F821)